src/main/java/org/springframework/data/couchbase/core/ReactiveCouchbaseTemplate.java

```diff
@@ -198,4 +202,32 @@ public QueryScanConsistency getConsistency() {
 		return scanConsistency;
 	}
 
+	/**
+	 *
+	 * @return the AttemptContextReactive for the transaction
+	 */
+	public AttemptContextReactive getCtx() {
```

Comment:
do those methods have to be public? (or can they be package private or completely private)

Reply:
package private works